README.md

```diff
@@ -11,6 +11,26 @@ pip install can_ada
 Binary wheels are available for most platforms. If not available, a
 C++17-or-greater compiler will be required to build the underlying Ada library.
 
+##  WHATWG URL compliance
+
+Unlike the standard library's `urllib.parse` module, this library is compliant with the WHATWG URL specification.
+
+```python
+urlstring = "https://www.GOoglé.com/./path/../path2/"
+import can_ada
+# prints www.xn--googl-fsa.com,
+# the correctly parsed domain name according to WHATWG
+print(can_ada.parse(urlstring).hostname)
+# prints /path2/
+# the correctly parsed pathname according to WHATWG
+print(can_ada.parse(urlstring).pathname)
+import urllib
+#prints www.googlé.com
```

Comment:
For my own sanity, can you add a space here and below so it's consistent ;)

Reply:
Done.